Sources/Commands/SwiftTool.swift

```diff
@@ -387,6 +387,7 @@ public class SwiftTool<Options: ToolOptions> {
                 // Terminate all processes on receiving an interrupt signal.
                 processSet.terminate()
 
+              #if !os(Windows)
```

Comment:
Can you re-arrange this better? Maybe `#if os(Windows)` .... `#elseif os(macOS) .... #else`. Some amount of duplication is fine if it leads to a better code readability.

Reply:
Fair point, should be a bit more readable now.